proposals/open-source-subsidy-program.md

```diff
@@ -0,0 +1,90 @@
+
+Therefore, our primary goal in this study is to calculate the "development activity threshold", which the program should use as a metric to standardize the subsidy approval process.
+
+We can summarize the study areas as the following:
+
+**Development activity threshold**: Study existing projects to see how much development activity it requires to build software on average—for example, lines of code, number of releases or commits, backlog changes. Inspecting existing repositories can help us determine an objective threshold.
+
+**Outline of the program**: Depends on the extensiveness, investigate the program's other aspects, such as:
+- The cost-benefit ratio predictions
+- High-level roadmap of the program with different phases
+- Effective subsidy usage strategies for maximizing the impact
+- Other policy or implementation challenges
+
+### Due diligence
+
+<!-- Please answer the following due diligence questions; it's okay to answer "N/A" if you don't know yet. -->
+
+1. **What related work has already been done in this area?**
+- [Open source software subsidies and network compatibility in a mixed duopoly](https://ideas.repec.org/a/ebl/ecbull/eb-13-00806.html) by Mourad Zeroukhi & Thierry Pénard focuses on using different subsidy strategies to increase open-source software usage.
+- [Study about the impact of open source software and hardware on technological independence, competitiveness and innovation in the EU economy](https://digital-strategy.ec.europa.eu/en/library/study-about-impact-open-source-software-and-hardware-technological-independence-competitiveness-and) by OpenForum Europe suggests subsidizing open-source software production, especially related to economic or social value.
+2. **How is this proposal innovative -- what distinguishes it from other related work?** No well-known study provides a detailed framework for implementing a subsidy program and tackling its practical challenges.
+3. **Who is your doer -- who will execute the proposed work?** It may depend on the extensiveness of the study. I'm willing to take responsibility for the execution. However, the proposal contains multiple areas, and collaborating with professional research organizations can be ideal.
+4. **How might this work be sustained long-term after an initial seed grant?** Providing the framework for this subsidy program should be a one-time study. The related government agencies should carry out the program's implementation if any find it feasible.
+
+### Resources needed
+
+A grant to allow us to carry out this study. It's possible to split the research into smaller focus areas; hence the amount could differ based on the extensiveness of the study. Providing contacts from existing research organizations would also be helpful.
```

Comment:
What exactly would this study fund? Such as researchers, or the actual money to be disbursed through subsidies, etc.?

Reply:
I thought to use the fund only for the researchers and the study itself.

On the other hand, we could also consider running a small pilot program to test the idea in practice once the study is over.

At the end of the study, we should have a framework/guidelines for running such a subsidy program. Once/if the government implements the program, they should allocate the resources and distribute the subsidy to the companies.